.github/ISSUE_TEMPLATE/service_request.md

```diff
@@ -3,9 +3,9 @@ name: Service request
 about: Suggest Rich Presence support for a website
 ---
 
-**Prerequisites and essential questions**
-<!--- Required, please answer the following questions as honestly as possible, not checking a question counts as "No". -->
+**Prerequisites and essential questions** <!--- Required, please answer the following questions as honestly as possible by changing the "[ ]" to "[x]" or by marking it after creating the issue (easier), not checking a question counts as "No". -->
 * [ ] Is it a popular site?
+* [ ] Is the website older than 2 months? <!--- It is necessary for the website to be older than 2 months. -->
```

Comment:
Maybe add like a link of where you can check?

Reply:
It's us who check lol

Reply:
They can do something aswell... xD

Reply:
I mean if it's not a site that's yours, or of a friend then its promotion and works and it has reached you somehow the same way it may have reached a lot others which means it may be quite popular and worthy of being added, and if it belongs to you then you already know when it was created.